### ISSUE TITLE
DLNA DMC: after send  JSON-RPC API 'Player.Open' image goes on top and can't watch more videos

### ISSUE THREAD

**Issue by Barabba11** [xbmc/xbmc#25443]
### Describe the bug
Hi, I'm using Kodi control Node Red module https://flows.nodered.org/node/node-red-contrib-kodi to send DLNA content to a TV running Kodi 21 by port 9090 and http transport, all working fine but when I load a picture I can't watch more videos, the videas are running only in background, I can hear audio, but the image still there. To quit from the picture player I need to press exit/back, then again fain a way to disappear the menu. The problem is the same for Windows and Android platform, so I suppose is a common problem.

## Expected Behavior
In my opinion trigger a video over a picture should mean close the picture before it, so I consider this a bug.
Anyway I'm asking if there is an overcome for that, any JSON-RPC API command I can send to Kodi to force the picture close and watch the video. 
Since this I can overcome this issue by Yatse (on port 8080) pressing "back" then pressing "fullscreen", I'm asking how can I reproduce eventally these Yatse commands using Node Red, if it is possible to send those commands by  JSON-RPC API, thank you.

## Actual Behavior
When I open a picture I can see only pictures, no more videos (I can only hear them, they are running in background)

## Possible Fix
Close automatically the photo viewer process when a video is called to be load

### To Reproduce
Enable Kodi control from external devices, load an http server or use shared folder ( samba protocol) and send  JSON-RPC API 'Player.Open' command to kodi, with Node red I'm using this function:

```
return {
    payload:
    {
        "cmd": "Player.Open",
        "args": {
            "item": {
                "file":"http://192.168.101.100/1.jpg"
            }
        }
    }
};
```

### Debuglog
no reasons to upload it, all is working
### Screenshots 
description is ehough to understand
## Additional context or screenshots (if appropriate)
no
### Your Environment
Kodi 21 on Windows and Android, probably the same problem for all platforms

Thank you

**Comment by PatK9** [comment]
[Solved](https://forum.kodi.tv/showthread.php?tid=378066&pid=3203178#pid3203178) by user

**Comment by Barabba11** [comment]
Hi, this is not solved, I've just find a remedy to temporary solve my problem. I think that Kodi should quit from image viewer if a video is loaded, why keep the image on monitor and just listen the audio of the video?

**Comment by wutschel** [comment]
I am not aware the use case described in bug report was behaving any different since at least Kodi 17. There are 3 players (0 = music, 1 = video and 2 = pictures) working on 3 playlists (same ids). If you want to stop a running picture playback, you just send Player.Stop with playlist 2. I did not test to send this when no picture is shown, but I imagine you could just simply send this before starting any other playback.

**Comment by Barabba11** [comment]
Thank you [wutschel](https://github.com/wutschel) for your kind info, can you kinly how you would add "playlist:2" or id:2 to the JSON I'm sending to Kodi? it would be interesting to give a try.

I've discovered another bug: Playing a directory wth pictures only will start slideshow, great, but if I play then a video it goes in background (expectable, as I described already) but when the next slide appearing (next picture on folder loaded) the background video is interrupted with the popup error,  I can see it when I quit from picture viewer. In my opinion this is a bug.

**Comment by wutschel** [comment]
Should be looking like this.

```
"method": "Player.Stop", "args": {"playerid": 2}
```

**Comment by Barabba11** [comment]
thank you, works. It stops also the videos quitting from video player

**Comment by Barabba11** [comment]
another bug, when a directory of images (only) is sent randomize function is not working, they will be showed always with the same order

**Comment by wutschel** [comment]
I guess you're loading this issue with too many different topics. Your main concern was that pictures/slideshows always go on top. Now you have an option to stop this when starting any other playback. So, workaround for your problem is available. I am not even sure, if this behaviour was made for a purpose.

In general you will find that slideshows are problematic, you just mentioned the randomize -- which btw works (just tried it again), but cannot be switched off w/o starting a new slideshow with Player.Open path/randomize/recursive again. There are more glitches in the details of the playlist handling which I will not go into details and also do not bother to raise an issue for as this will not catch any interest.

**Comment by Barabba11** [comment]
I don't want to overload, but I guess it's connected with image player, maybe I have to open another issue, I'm gonna do it, randomize doesn't work at all

**Comment by Barabba11** [comment]
I hope this issue can be solved in the next version of Kodi.
For now sending commands to stop player, then open video, but each time I see the kodi interface in background, that's not so beautiful, any chances to change the background to black screen?
